Source/WebCore/page/DOMTimer.cpp

```diff
@@ -404,9 +408,9 @@ Seconds DOMTimer::intervalClampedToMinimum() const
     return interval;
 }
 
-std::optional<MonotonicTime> DOMTimer::alignedFireTime(MonotonicTime fireTime) const
+std::optional<MonotonicTime> ScriptExecutionContext::alignedFireTime(bool hasReachedMaxNestingLevel, MonotonicTime fireTime) const
```

Comment:
Nit - is this intentionally implemented in `DOMTimer.cpp` instead of `ScriptExecutionContext.cpp`?

Reply:
Yes, it uses constants defined earlier in this file. To be cleaned up later.